docs/build-instructions/windows.md

```diff
@@ -31,8 +31,14 @@ Note: If you use cmd or Powershell instead of Git Shell, use a backslash instead
 These instructions will assume the use of Git Shell.
 
 ### `script/build` Options
-  * `--install-dir` - Creates the final built application in this directory.
-  * `--build-dir` - Build the application in this directory.
+  * `--install-dir` - Creates the final built application in this directory. Example (trailing slash is optional):
+```bash
+./script/build --install-dir Z:\Some\Destination\Directory\
```

Comment:
Thanks, I'll try to remember this bit, I read in contribution guidelines but forgot.

Is that something one should place in each commit message that does not touch code? Or should that be inserted in PR title? PR description?